snapshot/values.yaml

```diff
@@ -2,10 +2,13 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-#PlateRecognizer token and license
+# PlateRecognizer token and license - alternatively you can use `existingSecret`
 TOKEN:
 LICENSE_KEY:
 
```

Comment:
What is the use-case for this? AFAIK the container cannot start without TOKEN and LICENSE_KEY, they are not optional

Reply:
Usecase is using e.g. [Vault](https://www.vaultproject.io/) and [External-Secrets](https://external-secrets.io/latest/) to create the secret inside the cluster outside of the scope of this helm chart.

In our case our motivation for doing so is that we consider these values confidential and don't want to commit them along with our values file in our infrastructure-as-code setup. Rather we generate/keep them in our vault and generate configuration via `extraDeploy` to fetch them from there and create the equivalent secret that would otherwise be created from these values.

Reply:
@tbalzer for the container to start, it requires the TOKEN and LICENSE_KEY env variables to be present and valid.  

> we consider these values confidential and don't want to commit them

For this, we recommend passing the 2 values in the commands like so:
```shell
helm install platerec-stream stream/ --set TOKEN=<MY_TOKEN> --set LICENSE_KEY=<LICENSE_KEY>
helm upgrade platerec-stream stream/ --set TOKEN=<MY_TOKEN> --set LICENSE_KEY=<LICENSE_KEY>
```
The update of name to `platerec-snapshot` is okay with me. If you get rid of the new secret should be all good.

@marcbelmont any other thoughts are appreciated

Reply:
I know we could use `--set` but this would also happen inside our commited code in an infrastructure-as-code setup.
Yes, this could be solved via ENV vars configured in our CI system, but a much more powerful and configurable way is using e.g. vault (in our concrete case) or AWS Secrets Manager and then External-Secrets or similar services.